packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -473,6 +473,7 @@ password.complexity.minAgeMinutes = 》Åţ ļéåšţ ฐโ {0} ɱîñûţé
 password.complexity.minAgeHours = 》Åţ ļéåšţ ฐโ {0} ĥöûŕ❨š❩ ɱûšţ ĥåṽé éļåþšéð šîñçé ýöû ļåšţ çĥåñĝéð ýöûŕ þåššŵöŕð· Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 password.complexity.minAgeDays = 》Åţ ļéåšţ ฐโ {0} ðåý❨š❩ ɱûšţ ĥåṽé éļåþšéð šîñçé ýöû ļåšţ çĥåñĝéð ýöûŕ þåššŵöŕð· ฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 password.reset.verification = 》Ṽéŕîƒý ŵîţĥ öñé öƒ ţĥé ƒöļļöŵîñĝ ƒåçţöŕš ţö ŕéšéţ ýöûŕ þåššŵöŕð· ئ䀕ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+password.reset.revokeSessions = 》Šîĝñ ɱé öûţ öƒ åļļ öţĥéŕ ðéṽîçéš· Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
This also already exists in master, only adding here to prevent failure in tests